public/index.js

```diff
@@ -0,0 +1,13 @@
+$('.add-response').on('click', function (e) {
+  e.preventDefault();
+
+  var responses = $('.response-fields form #inputs');
+  var countable = $('.response-fields form .response');
+  var newResponse = `<input placeholder="Enter response type" name="inputs[${countable.length}]" class="response">`
```

Comment:
Without Babel, template strings are only supported in the most recent browsers and will break in older browsers.